docs/capability-types.md

```diff
@@ -0,0 +1,130 @@
+This is a full list of capability types with their properties, entities and units. See the [general information about capabilities](fixture-format.md#capabilities).
+
+### Possible entities and keywords
+
+| Entity | Allowed Units | `-100%` | `-1%` | `0%` | `1%` | `100%`
```

Comment:
1. Maybe explain a bit about the keywords first?
2. Should we add allowed ranges for each entity here, or are they dependent on the capability type?
3. Please format the table in the Markdown source as well. It may be a bit more effort but helps reviewing (makes switching between source and preview unnecessary).

Reply:
1. Good point

Reply:
Done.